maven-enforcer-plugin/src/it/projects/require-upper-bound-dependencies-managed_failure/verify.groovy

```diff
@@ -21,9 +21,9 @@ File buildLog = new File( basedir, 'build.log' )
 assert buildLog.text.contains( 'Rule 0: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:' )
 
 def message = 'Failed while enforcing RequireUpperBoundDeps. The error(s) are [' + LS+
-'Require upper bound dependencies error for org.apache.maven.plugins.enforcer.its:menforcer146-x:1.1 paths to dependency are:' + LS+
+'Require upper bound dependencies error for org.apache.maven.plugins.enforcer.its:menforcer146-x:1.1 [compile] paths to dependency are:' + LS+

```

Comment:
The word compile looks weird here.

Reply:
Yes, adding the scope there it does look out of place.

Reply:
modified, rebased and squashed.